core-web/apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.scss

```
@@ -30,14 +30,14 @@
             display: block;
             min-height: 100%;
             outline: none;
-            padding: $spacing-3 $spacing-8;
+            padding: $dot-editor-size calc(4 * $dot-editor-size);
```

Since this is sass

```suggestion
            padding: $dot-editor-size (4 * $dot-editor-size);
```

Maybe?

---

I'll do it.